### PR TITLE
upload: Spherical harmonics method for 1d task.

### PR DESCRIPTION
**Pull request by bridgeArchitect**
**A one line, short summary of intent.**

Links to issues such as "Fixes #XXX" or "Addresses #XXX".

A longer description and justification for the changes.

Raise any areas of concern.

A comment will appear below when the live site is built. This normally
takes a few minutes.

### CODE REVIEW

**Comment by pfhub** [comment]
@bridgeArchitect, the new version of the PFHub website is available at https://random-cat-1490.surge.sh

**Comment by wd15** [comment]
@bridgeArchitect: thanks so much for the upload! I'll fix a few things up and push to your branch and then I'll let you check it before merging.

**Comment by pfhub** [comment]
@bridgeArchitect, the new version of the PFHub website is available at https://random-cat-1490.surge.sh

**Comment by bridgeArchitect** [comment]
Dear Daniel Wheeler,

Thank you for your upload and modification. Now I can see
website with new information (simulation using spherical 
harmonics method).

My name is Artem Tomilo. I am the second year student of master's 
degree in TU Dresden from Ukraine. A university professor gave me 
the task of simulating a 1d problem using spherical harmonics instead of 
standard finite element or finite difference methods. I could do this using the 
Julia programming language.

It was my pleasure to solve the differential equation in partial derivatives using 
a new method. Thank you for this benchmark. My mood has improved significantly. 
This is one of the first personal events for me in the field of science and technology 
after the beginning of the war in my Motherland.

I'd like to know answer to the following question:

Is it possible to upload this data to the main site https://pages.nist.gov/pfhub/? 
Or will they be stored only on the site https://random-cat-1490.surge.sh/?

Best wishes,
Artem Tomilo

**Comment by tkphd** [comment]
Hi Artem, welcome! And thank you very much for submitting a solution to the spherical problem using Julia. It is delightful to see new approaches to solving the benchmarks, and I'm glad that writing it was a valuable distraction for you. I hope that your family and loved ones are all right.

After Daniel fixes some things in the auto-generated `meta.yaml` for your submission and you have verified that the way it looks on the preview site <https://random-cat-1490.surge.sh/>, this pull request will be merged, the preview site will shut down, and your results will be visible alongside everyone else's on the main site <https://pages.nist.gov/pfhub/>.

**Comment by bridgeArchitect** [comment]
Hi Trevor Keller,

With this message, I approve representation of sent data on  https://random-cat-1490.surge.sh/.
Now I'm waiting for a merge with the main branch for the site https://pages.nist.gov/pfhub/.

Best wishes,
Artem

**Comment by pfhub** [comment]
@bridgeArchitect, the new version of the PFHub website is available at https://random-cat-1490.surge.sh

**Comment by wd15** [comment]
@tkphd, @guyer: please review.

Travis CI failure is independent of this PR and fixed in #1485

**Review by wd15 — APPROVED** [review]

**Comment by bridgeArchitect** [comment]
Hi Daniel Wheeler,

Thank you for approving pull request.

Best wishes,
Artem

**Review by tkphd — APPROVED** [review]
Looks good to me. Interesting that the free energy evolution runs parallel to "the pack," higher than most but not all.

**Comment by tkphd** [comment]
@wd15 consider adding [SpeHarSol.jl](https://gitlab.com/bridgearchitect/speharsol) to the list of known codes?

**Comment by bridgeArchitect** [comment]
I agree with your opinion that time evolution is "higher" than most launches. I wrote it in the report for my university where I am studying.

**Comment by pfhub** [comment]
@bridgeArchitect, the new version of the PFHub website is available at https://random-cat-1490.surge.sh

File: _data/simulations/spheharm_1d_bridge/meta.yaml
Comment (guyer) [inline _data/simulations/spheharm_1d_bridge/meta.yaml:16]: @bridgeArchitect: In the renderings, e.g., https://gitlab.com/bridgearchitect/speharsol/-/blob/main/results/sphere1000.png, there is a thin wedge missing. I assume this is a rendering artifact and not an actual disclination in the mesh?

**Review by guyer — APPROVED** [review]

File: _data/simulations/spheharm_1d_bridge/meta.yaml
Comment (bridgeArchitect) [reply _data/simulations/spheharm_1d_bridge/meta.yaml:16]: Dear Jonathan Guyer,

Yes, it is correct. This artifact is caused by a feature of the library Plots.jl in the programming language Julia.

Best wishes,
Artem Tomilo

**Comment by wd15** [comment]
> @wd15: https://pages.nist.gov/pfhub/simulations/display/?sim=spheharm_1d_bridge, linked from the "Table of 1d.1 benchmark result uploads" goes to https://pages.nist.gov/pfhub/simulations/notfound/?sim=spheharm_1d_bridge .

That's a good spot. The table is no longer using a relative link, which it probably should be. It should be pointing at,

https://random-cat-1490.surge.sh/simulations/display/?sim=spheharm_1d_bridge

The display page appears to be empty for some reason. I'll probably deal with these issues in separate PRs.

**Comment by wd15** [comment]
> > @wd15: https://pages.nist.gov/pfhub/simulations/display/?sim=spheharm_1d_bridge, linked from the "Table of 1d.1 benchmark result uploads" goes to https://pages.nist.gov/pfhub/simulations/notfound/?sim=spheharm_1d_bridge .
> 
> That's a good spot. The table is no longer using a relative link, which it probably should be. It should be pointing at,
> 
> https://random-cat-1490.surge.sh/simulations/display/?sim=spheharm_1d_bridge
> 
> The display page appears to be empty for some reason. I'll probably deal with these issues in separate PRs.

Just following up, this is an issue on surge. It isn't loading some javascript libraries. For example, this is broken,

https://random-cat-1490.surge.sh/simulations/display/?sim=moose_1d_guan

but is fine on the main page,

https://pages.nist.gov/pfhub/simulations/display/?sim=moose_1d_guan

Regardless, all the individual landing pages need to be switched  to using notebook as well. Another project. I'm merging this now.

**Pull request merged**
